packages/hoppscotch-app/helpers/import-export/import/openapi/exampleV3.ts

```diff
@@ -0,0 +1,122 @@
+import { OpenAPIV3 } from "openapi-types"
+import { pipe } from "fp-ts/function"
+import * as A from "fp-ts/Array"
+import * as O from "fp-ts/Option"
+
+type SchemaType =
+  | OpenAPIV3.ArraySchemaObjectType
+  | OpenAPIV3.NonArraySchemaObjectType
+
+type PrimitiveSchemaType = Exclude<SchemaType, "array" | "object">
+
+type PrimitiveRequestBodyExampleType = string | number | boolean | null
+
+type RequestBodyExampleType =
+  | PrimitiveRequestBodyExampleType
+  | Array<RequestBodyExampleType>
+  | { [name: string]: RequestBodyExampleType }
+
+const isSchemaTypePrimitive = (
+  schemaType: SchemaType
+): schemaType is PrimitiveSchemaType =>
+  !["array", "object"].includes(schemaType)
+
+const getPrimitiveTypePlaceholder = (
+  primitiveType: PrimitiveSchemaType
+): PrimitiveRequestBodyExampleType => {
+  switch (primitiveType) {
+    case "number":
+      return 0.0
+    case "integer":
+      return 0
+    case "string":
+      return "string"
+    case "boolean":
+      return true
+  }
+}
+
+// Use carefully, call only when type is primitive
+// TODO(agarwal): Use Enum values, if any
+const generatePrimitiveRequestBodyExample = (
+  schemaObject: OpenAPIV3.NonArraySchemaObject
+): RequestBodyExampleType =>
+  getPrimitiveTypePlaceholder(schemaObject.type as PrimitiveSchemaType)
+
+// Use carefully, call only when type is object
+const generateObjectRequestBodyExample = (
+  schemaObject: OpenAPIV3.NonArraySchemaObject
+): RequestBodyExampleType =>
+  pipe(
+    schemaObject.properties,
+    O.fromNullable,
+    O.map(
+      (properties) =>
+        Object.entries(properties) as [string, OpenAPIV3.SchemaObject][]
+    ),
+    O.getOrElseW(() => [] as [string, OpenAPIV3.SchemaObject][]),
+    A.reduce(
+      {} as { [name: string]: RequestBodyExampleType },
+      (aggregatedExample, property) => {
+        aggregatedExample[property[0]] =
+          generateRequestBodyExampleFromSchemaObject(property[1])
+        return aggregatedExample
+      }
+    )
```

Comment:
We have a function pre-defined for this.
```suggestion
    tupleToRecord // import { tupleToRecord } from "~/helpers/functional/record"
```